### DISCUSSION TITLE
Haven't received Luna to eth chain after sending from Solana

### DISCUSSION THREAD

**Discussion by love2bluff** [wormhole-foundation/wormhole#272] [Problems / bugs] [answered]
Hi, I used Wormhole to transfer wLuna tokens back from Solana to Ethereum but received WWT tokens instead of Luna. Is it some sort of wrapped token i need to unwrap to receive Luna on eth? No Luna arrived to my MM wallet.

Tx - https://explorer.solana.com/address/9Ho9MMdST6zwfNdNpKFSyJA4g1dhgiB5AYmG4bAGrm4C
Screenshot from Debank transactions history - https://gyazo.com/67f71ed6f67e23ee7c4d600a387529f4

**Comment by love2bluff** [comment]
Ok, first I was told in Solana discord to post this into the Issue section. Now it has been moved to discussions for some reason and still no response.

**Reply by love2bluff** [reply]
Here is WWT tx https://etherscan.io/tx/0x280df2e65be8a5950b2848064e65d9dba245bac90e0e63cf82d756785e616d4b

**Comment by hendrikhofstadt** [comment, marked as answer]
You transferred renLUNA. The Wormhole can't unwrap REN tokens. You now have wrapped renLUNA on Ethereum. You need to transfer them back to Solana and then use the ren Bridge.

**Reply by love2bluff** [reply]
How can I do that? 
UI doesn't show any WWT tokens in my Eth wallet
Screenshot - https://gyazo.com/e3f19dbd6bdf351e0328bf4fd26fa5ea

**Reply by leoluk** [reply]
We don't maintain that UI, please file an issue here and request wormRenBTC added to the token list: https://github.com/solana-labs/oyster/issues
